packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -497,3 +495,38 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
     ],
   };
 }
+
+function getCacheSettings(
+  wco: WebpackConfigOptions,
+  supportedBrowsers: string[],
+): WebpackOptionsNormalized['cache'] {
+  if (persistentBuildCacheEnabled) {
+    const packageVersion = require('../../../package.json').version;
+
+    return {
+      type: 'filesystem',
+      cacheDirectory: findCachePath('angular-webpack'),
+      maxMemoryGenerations: 1,
+      // We use the versions and build options as the cache name. The Webpack configurations are too
+      // dynamic and shared among different build types: test, build and serve.
+      // None of which are "named".
+      name: createHash('sha1')
+        .update(NG_VERSION.full)
+        .update(packageVersion)
+        .update(wco.projectRoot)
+        .update(JSON.stringify(wco.tsConfig))
+        .update(JSON.stringify(wco.buildOptions))
+        .update(supportedBrowsers.join(''))
+        .digest('base64'),
+    };
+  }
+
+  if (wco.buildOptions.watch) {
```

Comment:
```suggestion
  if (wco.buildOptions.watch && !cachingDisabled) {
```
To maintain existing `NG_BUILD_CACHE` behavior.

Reply:
Good catch